qulice-maven-plugin/src/main/java/com/qulice/maven/DefaultMavenEnvironment.java

```diff
@@ -238,10 +239,11 @@ public boolean exclude(final String check, final String name) {
 
     @Override
     public Collection<String> excludes(final String checker) {
-        return Collections2.transform(
+        final Collection<String> excludes = Collections2.transform(
```

Comment:
@vkuchyn shouldn't the fix be in https://github.com/teamed/qulice/blob/6ef5d1056b0ea93144f7d99cfbd4fc829a0c5e9b/qulice-maven-plugin/src/main/java/com/qulice/maven/DependenciesValidator.java#L66?